pom.xml

```diff
@@ -216,6 +217,64 @@
     </build>
 
     <profiles>
+		<profile>
+			<id>rpm</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>rpm-maven-plugin</artifactId>
+						<version>2.1.5</version>
+						<executions>
+							<execution>
+								<id>generate-rpm</id>
+								<goals>
+									<goal>rpm</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<group>Applications/Internet</group>
+							<packager>Confluent Packaging</packager>
+							<license>${licenses.name}</license>
+							<needarch>noarch</needarch>
+							<targetOS>linux</targetOS>
+							<requires>
+								<require>confluent-common &gt;= ${project.version}</require>
```

Comment:
Thought about that but I wasn't sure how strictly we would want to adhere to that. For the top level RPM it is an issue. For the individual components not so much. In fact that is currently how you get a way with installing the exact value you want. By install each individual rpm one by one. 

I'll let you make the final verdict though because I plan on adding this to all the maven derived projects once you give it the thumbs up :)